features/facebookImport/src/styles.css

```diff
@@ -47,6 +47,10 @@
     color: var(--color-text-light);
 }
 
+button {
+    border: none;
+}
```

Comment:
You're completely right. I added this as a quick fix because the button's styling in the facebookImporter broke 😕 But I see it's something that it's standard in all the features.